llmfoundry/models/layers/llama_attention_monkeypatch.py

```diff
@@ -9,7 +9,8 @@
 from typing import Callable, Optional, Tuple
 
 import torch
-import torch.functional as F
+import torch.nn.functional as F
```

Comment:
```
>>> import torch.functional as F
>>> F.linear
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'torch.functional' has no attribute 'linear'
```

Was this code ever used / still maintained? If not, we should remove completely.

Reply:
woah, thats on me, thanks for catching. The file is copied and modified from the huggingface file. Somehow I must have copied the import statement incorrectly. `F` is only used on the `if self.config.pretraining_tp > 1:` codepaths, and the llama models uploaded to huggingface all have `self.config.pretraining_tp == 1:`. So that code path has not been tested within this repo, but is part of the upstream huggingface implementation, so I would be in favor of leaving it in even though it is currently untested.